hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -665,6 +665,7 @@ public MemberImpl getMember(String uuid) {
         return null;
     }
 
+    @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_NO_SIDE_EFFECT")
```

Comment:
`values()`, `keySet()` and `values()` are used to fill a cache. Should be fine to ignore the return values.